extensions/widgets/treeview/treeview.lcb

```diff
@@ -251,6 +251,24 @@ metadata sortType.editor is "com.livecode.pi.enum"
 metadata sortType.options is "text,numeric"
 metadata sortType.label is "Sort type"
 
+/**
+Name: pathDelimiter
+
+Type: property
+
+Syntax: set the pathDelimiter of <widget> to <pType>
+Syntax: get the pathDelimiter of <widget>
+
+Summary: Manipulets the separator between the elements of the array viewer
+
+Description:
+Use the <pathDelimiter> property as the separator between the elements describing
+a path in the array viewer of the property inspector.
```

Comment:
Instead of `array viewer in the property inspector` here, just say `tree view widget` - the widget can be used as a general object too.